lib/widgets/post.dart

```diff
@@ -419,12 +420,63 @@ class PostWidget extends HookWidget {
               postImage()
             else if (post.post.url != null && post.post.url.isNotEmpty)
               linkPreview(),
-            if (post.post.body != null)
-              // TODO: trim content
+            if (post.post.body != null && fullPost)
               Padding(
                   padding: const EdgeInsets.all(10),
                   child:
                       MarkdownText(post.post.body, instanceHost: instanceHost)),
+            if (post.post.body != null && !fullPost)
+              LayoutBuilder(
+                builder: (context, constraints) {
+                  final span = TextSpan(
+                    text: post.post.body,
+                  );
+                  final tp = TextPainter(
+                    text: span,
+                    maxLines: 20,
```

Comment:
I know in the issue it said 20 lines, but it seems like it's a bit much. Could you change it to 10 lines?